proto/stride/stakeibc/tx.proto

```diff
@@ -210,35 +212,42 @@ message MsgResumeHostZone {
 message MsgResumeHostZoneResponse {}
 
 message MsgCreateTradeRoute {
-  string creator = 1;
+  option (cosmos.msg.v1.signer) = "authority";
+  option (amino.name) = "stride/x/stakeibc/MsgCreateTradeRoute";
+
+  // authority is the address that controls the module (defaults to x/gov unless
+  // overwritten).
+  string authority = 1 [ (cosmos_proto.scalar) = "cosmos.AddressString" ];
+
+  // The chain ID of the host zone
   string host_chain_id = 2;
-  string host_connection_id = 3;
-  string host_ica_address = 4;
-  string reward_chain_id = 5;
-  string reward_connection_id = 6;
-  string reward_ica_address = 7;
-  string trade_chain_id = 8;
-  string trade_connection_id = 9;
-  string trade_ica_address = 10;
-  string host_reward_transfer_channel_id = 11;
-  string reward_trade_transfer_channel_id = 12;
-  string trade_host_transfer_channel_id = 13;
-  string reward_denom_on_host = 14;
-  string reward_denom_on_reward = 15;
-  string reward_denom_on_trade = 16;
-  string target_denom_on_trade = 17;
-  string target_denom_on_host = 18;
-  uint64 pool_id = 19;
-  string min_swap_amount = 20 [
-    (cosmos_proto.scalar) = "cosmos.Int",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable) = false
-  ];
-  string max_swap_amount = 21 [
-    (cosmos_proto.scalar) = "cosmos.Int",
-    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
-    (gogoproto.nullable) = false
-  ];
+
+  // Connection IDs between stride and the other zones
```

Comment:
You are not including the stride_to_host_connection_id because we assume that while reward and trade chains might not be in the hostZones array, the "host zone" will be and so we can get the connection id (which is needed in ICA messages we send it) from the normal hostZone data structure?

Reply:
exactly!